orchestration/dagster_orchestration/hca_orchestration/tests/test_solids.py

```diff
@@ -45,12 +48,21 @@ def test_post_import_validate(self, mock_all_table_names: Mock, mock_duplicates:
             }
         }
 
-        result = execute_solid(base_post_import_validate, run_config=solid_config, mode_def=test_mode)
+        result = execute_solid(base_post_import_validate,
+                               run_config=solid_config, mode_def=test_mode)
         self.assertTrue(result.success)
-        expected_duplicate_issues = len(fake_table_names) * len(fake_duplicate_ids)
-        expected_file_ref_issues = len(fake_file_table_names) * len(fake_null_fileref_ids)
-        self.assertEqual(expected_duplicate_issues, result.output_value().duplicates)
-        self.assertEqual(expected_file_ref_issues, result.output_value().null_file_refs)
+        expected_duplicate_issues = len(
```

Comment:
this looks like the result of accidentally running `autopep8` without our custom line length config getting detected. i don't care a huge amount but i think we should revert it if it's not too much effort, since i think this level of indentation impairs readability a bit

Reply:
Fixed.